src/acl.c

```diff
@@ -822,7 +822,7 @@ robj *ACLDescribeUser(user *u) {
         if (selector->flags & SELECTOR_FLAG_ROOT) {
             res = sdscatfmt(res, "%s", default_perm);
         } else {
-            res = sdscatfmt(res, " (%s)", default_perm);
+            res = sdscatfmt(res, " \"(%s)\"", default_perm);
```

Comment:
i'm afraid this can affect / break some existing use cases (people parsing the ACL file or the ACL LIST output).
maybe we should add quotes only when the string contains some chars?
or maybe we should use something similar to `sdsneedsrepr` and `sdscatrepr`?

Reply:
I was waffling on whether I liked this solution, mostly for the reasons that were mentioned. The alternative is we can just fix the parsing logic and understand that the ")" that was encountered as part of a key pattern and not related to selectors. Which I still think I prefer as a solution for now.

Reply:
@oranagra I agree with your idea of quoting only if certain character appears (currently `)`) . If we want to quote the selectors irrespective of chars for 8.0 and beyond, we would need to revisit this.

@madolson As discussed offline, the solution you proposed seems to be unviable.